01week/datatypes.js

```diff
@@ -0,0 +1,186 @@
+  }
+  // check for String type
+  else if (typeof arg1 === "string") {
+    console.log(arg1 + " is type String");
+  }
+  // begin checking FALSY data types
+  // check for null type
+  else if (arg1 === null) {
+    console.log(arg1 + " is type Null");
+  }
+  // check for Undefinded type
+  else if (arg1 === undefined) {
+    console.log(arg1 + " is type Undefined");
+  }
+  // check for NaN type using isNaN method
+  else if (isNaN(arg1)) {
+    console.log(arg1 + " is type NaN");
+  }
+};
+
+// check Boolean type
+checkDataType(true);
+// check Null type
+checkDataType(null);
+// check Undefinded type
+checkDataType(undefined);
+// check Number type
+checkDataType(31);
+// check NaN type
+// parseInt("string not a number") will return a value of NaN
+checkDataType(parseInt("string not a number"));
+// check String type
+checkDataType("Howdydoodee");
+
+console.log(); // blank line separator
+
+// 5. write a js function that adds two numbers together
+//
+// oneline function that returns the addition of two arguments
+
+const addTwoNumbers=(arg1, arg2)=> arg1 + arg2;
+
+console.log(addTwoNumbers(20, 40));
+
+console.log(); // blank line separator
+
+// 6. write a js function that runs only when 2 'things' (args) are true
+//
+// the wording of the problem says runs only when 2 things are "true" and not "truthy"
+// so I assume the expected arguments are boolean type
+//
+// check both args given to see if they === true, if so, print message
+// if 1 or more args !== true, do nothing
+
+const twoArgsTrue=(arg1, arg2)=>{
+  if (arg1 === true && arg2 === true) {
+    console.log("Both args are true");
+  }
+};
+// this should print a message
+twoArgsTrue(true, true);
+// this should not print a message
+twoArgsTrue(true, false);
+
+console.log(); // blank line separator
+
+// 7. write a js function that runs when 1 of 2 things are true
+//
+// the wording of the problem says runs only when 1 of 2 things are "true" and not "truthy"
+// so I assume the expected arguments are boolean type
+//
+// check both args, if 1 arg === true && 1 arg === false, then print message
+// any other case, do not run
+
+const oneTrueOneFalse=(arg1, arg2)=>{
+  if (arg1 == true && arg2 === false) {
+    console.log("arg1 is True, arg2 is False");
+  } else if (arg1 === false && arg2 === true) {
+    console.log("arg1 is False, arg2 is True");
+  }
+};
+// these should print a message
+oneTrueOneFalse(true, false);
+oneTrueOneFalse(false, true);
+// this should NOT print a message
+oneTrueOneFalse(true, true);
+
+console.log(); // blank line separator
+
+// 8. write a js function that runs when both 'things' (args) are not true
+//
+// the wording of the problem says runs only when both things are "false" and not "falsy"
+// so I assume the expected arguments are boolean type
+//
+// check both args given to see if === false, if so, print message
+// any other case, do not print message
+
+const twoArgsFalse=(arg1, arg2)=>{
+  if (arg1 === false && arg2 === false) {
```

Comment:
This evaluates if the arguments are the boolean false.I need a function that evaluates if both inputs evaluate to falsy